css/metabox-primary-category.css

```diff
@@ -0,0 +1,32 @@
+.wpseo-primary-term > label,
+.wpseo-is-primary-term {
+	font-weight: bold;
+}
+
+.wpseo-primary-term > .wpseo-make-primary-term,
+.wpseo-term-unchecked > .wpseo-make-primary-term {
+	display: none;
```

Comment:
The `.wpseo-make-primary-term` is conditionally shown when the term is not primary. I tested this with OS X voiceover and it speaks the `span`s.